src/request/ExtractInformationFromRequest.ts

```diff
@@ -97,10 +97,12 @@ function extractInformationFromBody(
 
     const contentType = getContentType(contentTypeFromHeader)
     switch (contentType) {
-        case ContentType.graphql:
-            return { query: bodyIsString ? body : JSON.stringify(body) }
-        case ContentType.json:
-            if (bodyIsString) {
+        case 'application/graphql':
```

Comment:
Can't this use the enum values anymore?

Reply:
No, because ContentType isn't an enum anymore but a string type. This is nonetheless type safe as the type only allows defined string values.